test-lib.sh

```diff
@@ -40,37 +62,67 @@ UNSTABLE_TESTS="${UNSTABLE_TESTS:-""}"
 function ct_cleanup() {
   ct_show_resources
   ct_show_results
+  ct_clean_app_images
 
   if [[ -z ${CID_FILE_DIR:-} ]]; then
     echo "The \$CID_FILE_DIR is not set. Container cleaning is to be skipped."
     exit "${TESTSUITE_RESULT:-0}"
   fi;
 
-  echo "Examining CID files in $CID_FILE_DIR"
+  # return 0 if container exists
+  function container_exists() {
```

Comment:
Why function in function? Can it be moved outside?

Reply:
Fixed in https://github.com/sclorg/container-common-scripts/pull/306/commits/988e2c479cd2b2a113d5b64a911fab7deadd3f74.